src/server/plugins/reactHtml/createRequestHtmlFragment.jsx

```diff
@@ -121,13 +128,23 @@ const createRequestHtmlFragment = async (request, reply, { createRoutes }) => {
         ? renderForStaticMarkup
         : renderForString;
 
+      const finishRenderTimer = startSummaryTimer(
+        ssrMetrics.reactRendering,
+        {
+          // strip query params and trailing slash
+          path: /^(.+?)\/?(\?.+)?$/.exec(request.url)[1],
```

Comment:
this is going to cause a large increase in memory :S (even after reducing the percentiles)
1 entry per percentile + 2 (sum and count) = 6 entries per path
```
# HELP oneapp_ssr_react_rendering_seconds time taken by React to render HTML
# TYPE oneapp_ssr_react_rendering_seconds summary
oneapp_ssr_react_rendering_seconds{quantile="0.1",path="/api/live/ws",renderMethodName="renderForString"} 0.003503204
oneapp_ssr_react_rendering_seconds{quantile="0.5",path="/api/live/ws",renderMethodName="renderForString"} 0.003503204
oneapp_ssr_react_rendering_seconds{quantile="0.9",path="/api/live/ws",renderMethodName="renderForString"} 0.003503204
oneapp_ssr_react_rendering_seconds{quantile="0.95",path="/api/live/ws",renderMethodName="renderForString"} 0.003503204
oneapp_ssr_react_rendering_seconds_sum{path="/api/live/ws",renderMethodName="renderForString"} 0.003503204
oneapp_ssr_react_rendering_seconds_count{path="/api/live/ws",renderMethodName="renderForString"} 1
oneapp_ssr_react_rendering_seconds{quantile="0.1",path="/oh",renderMethodName="renderForString"} 0.000447032
oneapp_ssr_react_rendering_seconds{quantile="0.5",path="/oh",renderMethodName="renderForString"} 0.000447032
oneapp_ssr_react_rendering_seconds{quantile="0.9",path="/oh",renderMethodName="renderForString"} 0.000447032
oneapp_ssr_react_rendering_seconds{quantile="0.95",path="/oh",renderMethodName="renderForString"} 0.000447032
oneapp_ssr_react_rendering_seconds_sum{path="/oh",renderMethodName="renderForString"} 0.000447032
oneapp_ssr_react_rendering_seconds_count{path="/oh",renderMethodName="renderForString"} 1
oneapp_ssr_react_rendering_seconds{quantile="0.1",path="/_/report/errors",renderMethodName="renderForString"} 0.000473052
oneapp_ssr_react_rendering_seconds{quantile="0.5",path="/_/report/errors",renderMethodName="renderForString"} 0.000473052
oneapp_ssr_react_rendering_seconds{quantile="0.9",path="/_/report/errors",renderMethodName="renderForString"} 0.000473052
oneapp_ssr_react_rendering_seconds{quantile="0.95",path="/_/report/errors",renderMethodName="renderForString"} 0.000473052
oneapp_ssr_react_rendering_seconds_sum{path="/_/report/errors",renderMethodName="renderForString"} 0.000473052
oneapp_ssr_react_rendering_seconds_count{path="/_/report/errors",renderMethodName="renderForString"} 1
```
While this would be useful data, I think the cost outweighs the benefit. I think operators would still get a lot of benefit without breaking out by path.